### ISSUE TITLE
Can't run in2csv with python 2.7

### ISSUE THREAD

**Issue by xavierbeynon** [wireservice/csvkit#682]
Followed the [installation guide from the docs](http://csvkit.readthedocs.io/en/0.9.1/install.html#developers).  Attempted to run tests using nosetests.

```
======================================================================
ERROR: Failure: AttributeError (type object 'Table' has no attribute 'monkeypatch')
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/Users/xavierbeynon/.virtualenvs/csvkit/lib/python2.7/site-packages/nose/loader.py", line 418, in loadTestsFromName
    addr.filename, addr.module)
  File "/Users/xavierbeynon/.virtualenvs/csvkit/lib/python2.7/site-packages/nose/importer.py", line 47, in importFromPath
    return self.importFromDir(dir_path, fqname)
  File "/Users/xavierbeynon/.virtualenvs/csvkit/lib/python2.7/site-packages/nose/importer.py", line 94, in importFromDir
    mod = load_module(part_fqname, fh, filename, desc)
  File "/Users/xavierbeynon/csvkit/csvkit/tests/test_convert/test_fixed.py", line 6, in <module>
    from csvkit.utilities.in2csv import In2CSV
  File "/Users/xavierbeynon/csvkit/csvkit/csvkit/utilities/in2csv.py", line 12, in <module>
    agatedbf.patch()
  File "/Users/xavierbeynon/.virtualenvs/csvkit/lib/python2.7/site-packages/agatedbf/__init__.py", line 12, in patch
    agate.Table.monkeypatch(TableDBF)
AttributeError: type object 'Table' has no attribute 'monkeypatch'
```

Any insight here?

**Comment by jpmckinney** [comment]
Hmm, did all the `pip` and `python` commands complete without errors?

**Comment by jpmckinney** [comment]
Fixed by eb8d113da3aefb0fac9dbdab1663160d15e5488f
